optuna_dashboard/ts/components/CompareStudies.tsx

```diff
@@ -0,0 +1,313 @@
+                          }
+                          history.push(getStudyListLink(next))
+                        } else {
+                          history.push(getStudyListLink([study.study_id]))
+                        }
+                      }}
+                      selected={
+                        selected.findIndex(
+                          (s) => s.study_id === study.study_id
+                        ) !== -1
+                      }
+                      sx={{
+                        display: "flex",
+                        flexDirection: "column",
+                        alignItems: "flex-start",
+                      }}
+                    >
+                      <ListItemText primary={study.study_name} />
+                      <Box
+                        sx={{
+                          display: "flex",
+                          flexDirection: "row",
+                          width: "100%",
+                        }}
+                      >
+                        <Typography
+                          color={theme.palette.grey.A400}
+                          sx={{ p: theme.spacing(0, 1) }}
+                        >
+                          {`# ${study.study_id}`}
+                        </Typography>
+                        <Chip
+                          color="primary"
+                          label={
+                            study.directions.length === 1
+                              ? `${study.directions.length} objective`
+                              : `${study.directions.length} objectives`
+                          }
+                          sx={{ margin: theme.spacing(0) }}
+                          size="small"
+                          variant="outlined"
+                        />
+                      </Box>
+                    </ListItemButton>
+                  </ListItem>
+                )
+              })}
+            </List>
+          </Box>
+          <Divider orientation="vertical" flexItem />
+          <Box
+            sx={{
+              flexGrow: 1,
+              overflow: "auto",
+              height: `calc(100vh - ${theme.spacing(8)})`,
+            }}
+          >
+            <Box sx={{ display: "flex", flexDirection: "row", width: "100%" }}>
+              <StudiesGraph studies={selected} />
+            </Box>
+          </Box>
+        </Box>
+      </AppDrawer>
+    </Box>
+  )
+}
+
+const StudiesGraph: FC<{ studies: StudySummary[] }> = ({ studies }) => {
+  const theme = useTheme()
+  const action = actionCreator()
+  const studyDetails = useRecoilValue<StudyDetails>(studyDetailsState)
+  const [logScale, setLogScale] = useState<boolean>(false)
+  const [includePruned, setIncludePruned] = useState<boolean>(true)
+
+  const handleLogScaleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setLogScale(!logScale)
+  }
+
+  const handleIncludePrunedChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setIncludePruned(!includePruned)
+  }
```

Comment:
Does this switch do anything with the current plots? For instance, for the optimization history plot, pruned trials are never plotted (since they don't have necessarily have any associated values). Not a strong opinion but we could remove/hide the toggle for now.

Reply:
Ok. That could be. Wait for @c-bata review.

Reply:
> Does this switch do anything with the current plots?

Yes. If `includePrunedTrial` is true, the latest intermediate value is used for the pruned trials like below:

![include-pruned](https://user-images.githubusercontent.com/5564044/221735635-3327e4b5-da88-45c9-9801-e6f82442a2c9.gif)

So I think we can leave this toggle.